src/app/main.jsx

```diff
@@ -26,19 +27,30 @@ class Main extends React.Component{
   }
 
   loadAssemblies (org) {
-    const classThis = this;
+    const thisClass = this;
     assems(this.error,{ooOrganization:org}, (assemObjs) => {
       this.props.setAssemblies(assemObjs , org);
       assemObjs.forEach(function(assem) {
```

Comment:
if you make this an arrow function, does it get the context (`this`) you want without having to assign it to a variable? Asking bc I don't know